### PR TITLE
Fixed bug with triviaURL string on getQuestion() for custom databases

### PR DESCRIPTION
**Pull request by MarkBrosche**

### CODE REVIEW

File: events/lastAvatarStandingTrivia/README.md [hunk 1]
```diff
@@ -1,15 +1,23 @@
 # Last Avatar Standing Trivia Game
 
 ### Release Notes:
+**February 15, 2019**
+<<<<<<< HEAD
```
Comment (zfox23) [inline events/lastAvatarStandingTrivia/README.md]: looks like you have a merge conflict in here?

File: events/lastAvatarStandingTrivia/README.md [hunk 2]
```diff
@@ -31,9 +39,14 @@ You can switch databases at any time, but to re-enter custom or miscellaneous da
 ### TO DO LIST
 
 1. Write up instructions on how to format a Google Sheets custom trivia database and Google script.
-2. Create an app loader that automatically loads the Host app in world.
+2. Create an app loader that automatically loads the Host app in-world.
 3. Condense color check scripts into one.
 4. Simplify html with button state-machine.
 5. Replace timeout on unloading the color check scripts with a callback when player removals are complete.
+<<<<<<< HEAD
```
Comment (zfox23) [inline events/lastAvatarStandingTrivia/README.md]: merge conflict?

File: events/lastAvatarStandingTrivia/clientScripts/trivia.html [hunk 3]
```diff
@@ -112,6 +112,7 @@ <h4>Choose Question Details (Optional) <br>& Click 'New Question'</h4>
             }
             function onScriptEventReceived(event) {
                 if (event === "newGame") {
+                    console.log("NEW GAME TIMES!!!");
```
Comment (zfox23) [inline events/lastAvatarStandingTrivia/clientScripts/trivia.html]: are these logs necessary?

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js [hunk 4]
```diff
@@ -25,7 +25,7 @@
         TEN_SECONDS_MS = 10 * ONE_SECOND_MS,
         ZONE_COLOR_INDEX = 19,
         HALF_MULTIPLIER = 0.5,
-        FIRST_WAIT_TO_COUNT_AVATARS = 1500,
+        FIRST_WAIT_TO_COUNT_AVATARS = 1000,
```
Comment (zfox23) [inline events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: why this change?

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js [hunk 5]
```diff
@@ -332,8 +332,9 @@
             try {
                 if (triviaURL === "") {
                     triviaURL = SECRETS.trivia_URL;
-                }
-                triviaURL = triviaURL + "category=" + GSHEET_TAB_NAME;
+                } else if (triviaURL.indexOf("category=") === -1) {
+                    triviaURL = triviaURL + "category=" + GSHEET_TAB_NAME;
```
Comment (zfox23) [inline events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: why is this necessary?

Comment (MarkBrosche) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: it was adding "category=GSHEET_TAB_NAME" to the end of the URL everytime getQuestion() was called.

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js (hunk 4 above)
Comment (MarkBrosche) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: Whoops, it's arbitrary, I tried messing with it to understand the senquencing of events at the end of the countdown.  1000 or 1500 doesn't actually make a difference in testing it out.

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js (hunk 5 above)
Comment (zfox23) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: right - but i don't understand why this new method of appending `"category=" + GSHEET_TAB_NAME` works. is `triviaURL` not actually a URL?

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js (hunk 4 above)
Comment (zfox23) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: OK. what if the delay were 0? would that still work?

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js (hunk 5 above)
Comment (MarkBrosche) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: our custom google database has multiple tabs and the way you access them is via that appendage.

File: events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js (hunk 4 above)
Comment (MarkBrosche) [reply events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js]: It does work, but the incorrect sound effect cuts off the end of the countdown music so it's nicer to have a lil buffer.
